### PR TITLE
Update motion_blinds.markdown

### PR DESCRIPTION
**Pull request by stefanhinz**
Added Heicko as another supported vendor/product.
Expanded the Connector app section with instructions on how to find the About page.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added Heicko brand and Heicko Smart Stick 1ST to the list of compatible devices.
- **Documentation**
	- Enhanced instructions for accessing the API key in the Connector app for better user guidance.

### CODE REVIEW

File: source/_integrations/motion_blinds.markdown [hunk 1]
```diff
@@ -94,7 +96,7 @@ In the official Bloc Blinds app go to settings (three bars > gear icon), go to t
 
 ### Connector app
 
-Click the about page of the connector app 5 times to get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)).
+Click the About page of the Connector app 5 times to get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)). The About page is in the left sidebar of the app, under Settings (gear icon). Selecting About opens the About page, which you can tap on 5 times to open a window with the API key.
```
Comment (c0ffeeca7) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
 To get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
 
  1. In the left sidebar of the app, open the **Settings** {% icon "mdi:gear-outline" %} (gear icon). 
  2. Select the **About** page of the Connector app 5 times. 
      - This opens a window with the API key.
```

File: source/_integrations/motion_blinds.markdown [hunk 2]
```diff
@@ -94,7 +96,11 @@ In the official Bloc Blinds app go to settings (three bars > gear icon), go to t
 
 ### Connector app
 
-Click the about page of the connector app 5 times to get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)).
+ To get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
```
Comment (c0ffeeca7) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
 To get the API key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @stefanhinz :+1:

**Comment by c0ffeeca7** [comment]
@starkillerOG, are you good with the changes?

File: source/_integrations/motion_blinds.markdown [hunk 3]
```diff
@@ -94,7 +96,11 @@ In the official Bloc Blinds app go to settings (three bars > gear icon), go to t
 
 ### Connector app
 
-Click the about page of the connector app 5 times to get the key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)).
+ To get the API key ([iOS app](https://apps.apple.com/us/app/connector/id1344058317), [Android app](https://play.google.com/store/apps/details?id=com.smarthome.app.connector)), follow these steps:
+ 
+  1. In the left sidebar of the app, open the **Settings** {% icon "mdi:gear-outline" %} (gear icon). 
+  2. Select the **About** page of the Connector app 5 times.
```
Comment (starkillerOG) [inline source/_integrations/motion_blinds.markdown]:
```suggestion
  2. Select the **About** page of the Connector app.
  3. Tap the screen 5 times while being on the **About** page. 
```

**Comment by starkillerOG** [comment]
@c0ffeeca7 yes, but I find the last step a bit unclear, you don't open and close the page 5 times, you just click the screen 5 times while beein om the About page.

Might just be me.

Maybe you can come up with better wording for this last step

Rest is good.

**Comment by c0ffeeca7** [comment]
> @c0ffeeca7 yes, but I find the last step a bit unclear, you don't open and close the page 5 times, you just click the screen 5 times while beein om the About page.
> 
> Might just be me.
> 
> Maybe you can come up with better wording for this last step
> 
> Rest is good.

Thank you for the fix!

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @stefanhinz and @starkillerOG :+1:

**Pull request merged**